install.py

```diff
@@ -421,6 +421,14 @@ def build_legion(
             + setup_py_flags,
             cwd=legion_python_dir,
         )
+    verbose_check_call(
```

Comment:
Should this be added to the `INSTALL_HEADERS` in https://github.com/StanfordLegion/legion/blob/master/runtime/runtime.mk#L1030?

Reply:
I guess we should check with Mike but I don't think we actually want to expose that header, though it's necessary for the core @lightsighter any thoughts?